src/TSTransformer/nodes/statements/transformImportEqualsDeclaration.ts

```diff
@@ -1,11 +1,20 @@
 import luau from "@roblox-ts/luau-ast";
 import { assert } from "Shared/util/assert";
 import { TransformState } from "TSTransformer";
+import { transformIdentifierDefined } from "TSTransformer/nodes/expressions/transformIdentifier";
 import { transformVariable } from "TSTransformer/nodes/statements/transformVariableStatement";
 import { createImportExpression } from "TSTransformer/util/createImportExpression";
 import { isSymbolOfValue } from "TSTransformer/util/isSymbolOfValue";
 import ts from "typescript";
 
+function transformEntityName(state: TransformState, node: ts.EntityName) {
+	return ts.isIdentifier(node) ? transformIdentifierDefined(state, node) : transformQualifiedName(state, node);
+}
+
+function transformQualifiedName(state: TransformState, node: ts.QualifiedName): luau.PropertyAccessExpression {
+	return luau.property(transformEntityName(state, node.left), node.right.text);
+}
```

Comment:
Are any of these used in other places in TS? It might be beneficial to move them to separate files, like some other generic node transforms currently are.

Reply:
It's used in type imports, JS Doc nodes, etc. nothing else we compile. I decided to keep them in this file since they create a recursive loop. Would be annoying to jump between files to try and understand it.

Reply:
They could both be in a separate file together, as they're not linked to importEqualsDeclaration specifically.

Reply:
If they ever get used by something else, we can move them out of this file. I don't expect that to ever happen.